builtin/logical/consul/path_roles.go

```diff
@@ -143,7 +152,6 @@ func (b *backend) pathRolesRead(ctx context.Context, req *logical.Request, d *fr
 	// Generate the response
 	resp := &logical.Response{
 		Data: map[string]interface{}{
-			"lease":            int64(roleConfigData.TTL.Seconds()),
 			"ttl":              int64(roleConfigData.TTL.Seconds()),
 			"max_ttl":          int64(roleConfigData.MaxTTL.Seconds()),
 			"token_type":       roleConfigData.TokenType,
```

Comment:
Any thoughts around removing "lease" from the response here? It returns the same data as "ttl", but lease is the legacy field name. The response currently provides that same value twice because it contains both key names.

Reply:
Opinions may vary here, but I think it's best to stick with our compatibility contract no matter how small the change. This would be considered API breaking, since a client could depend on it being present in the response. I would suggest for any field removal, we follow the formal deprecation process.

Reply:
Yes, that is true. We could leave "lease" here until we consider removing the entire field + the other potentially deprecated fields alongside it in the future. I don't have a reason to push for a quick removal. FWIW I would anticipate less friction with the lease field as it has been marked as deprecated since Vault release 1.0.0 (Dec 2018).

Reply:
Do we need to add an entry [here](https://www.vaultproject.io/docs/deprecation) to initiate the deprecation process for this field?

Reply:
The `Deprecated` field in `framework.FieldSchema` is only used by openapi. So unless clients used the [Vault OpenAPI generation](https://www.vaultproject.io/api-docs/system/internal-specs-openapi#sys-internal-specs-openapi), there is no guarantee that clients are aware of its deprecation.

Reply:
Based on our discussions today, I'm removing the deprecation tag from token_type and policy and the lease field as been added back to the role read response. That will be a separate PR after more feedback is received later this week. This PR instead now focuses only on the `consul_policies` parameter.